third_party/pyth/price-service/README.md

```diff
@@ -1,54 +1,58 @@
+These endpoints can be used to test integrations with Pyth Network:
 
-```
-docker buildx build -f Dockerfile.wasm -o type=local,dest=. .
-```
+| network | url                             |
+| ------- | ------------------------------- |
+| mainnet | https://xc-mainnet.pyth.network |
+| testnet | https://xc-testnet.pyth.network |
 
-Then, build the pyth_price_service docker container from the repo root:
+For production deployments, developers integrating with Pyth Network are **strongly encouraged** to host their own instance of the price service for maximum resilience and decentralization.
+By running an independent instance of this service, developers tap directly into Wormhole's peer-to-peer network to stream Pyth price updates.
+This peer-to-peer network has built-in redundancy and is therefore inherently more reliable than a centralized service operated by the PDA.
 
-```
-$ docker buildx build -f third_party/pyth/price-service/Dockerfile.price_service -t pyth_price_service .
-```
+## Wormhole Spy
 
-Run the spy_guardian docker container in TestNet:
+Price service depends on a Wormhole spy to stream Pyth messages from the Wormhole Network to it.
+[Spy](https://github.com/wormhole-foundation/wormhole/blob/main/node/cmd/spy/spy.go) is a component of a
+Wormhole guardian node that joins the Wormhole Network peer-to-peer network and listens to the Wormhole verified
+messages and streams the messages that are coming from certain emitters (e.g., Pyth data emitters) to its subscribers.
 
-```
-$ docker run --platform linux/amd64 --network=host ghcr.io/certusone/guardiand:v2.8.8.1 spy \
---nodeKey /node.key --spyRPC "[::]:7073" \
---bootstrap /dns4/wormhole-testnet-v2-bootstrap.certus.one/udp/8999/quic/p2p/12D3KooWBY9ty9CXLBXGQzMuqkziLntsVcyz4pk1zWaJRvJn6Mmt \
---network /wormhole/testnet/2/1
-```
+## Run
 
-Or run the spy_guardian docker container in MainNet:
-For the MainNet gossip network parameters, see https://github.com/wormhole-foundation/wormhole-networks/blob/master/mainnetv2/info.md
+To run Pyth price service, you need to have Wormhole spy running. You can simply run both price service
+and spy using provided mainnet and testnet docker compose files. To run the mainnet docker
+compose file run the following command:
 
 ```
-$ docker run --platform linux/amd64 -d --network=host ghcr.io/certusone/guardiand:v2.8.8.1 spy \
---nodeKey /node.key --spyRPC "[::]:7073" \
---bootstrap <guardianNetworkBootstrapParameterForMainNet> \
---network <guardianNetworkPathForMainNet> \
+docker-compose up -f docker-compose.mainnet.yaml
 ```
 
-Then to run the pyth_price_service docker container using a config file called `${HOME}/pyth_price_service/env` and logging to directory `${HOME}/pyth_price_service/logs`, do the
-following:
+The compose files use a public release of Pyth price service and spy. If you wish to change the
+price service, you need to build an image for using it first.
```

Comment:
build an image and then presumably edit the compose file to point to it? what are the steps to update the compose file?